examples/fullstack-one-example/index.ts

```diff
@@ -24,27 +23,40 @@ const $gql: GraphQl = Container.get(GraphQl);
 const $gs: GracefulShutdown = Container.get(GracefulShutdown);
 const $autoMigrate: AutoMigrate = Container.get(AutoMigrate);
 const $fs: FileStorage = Container.get(FileStorage);
+
 const $auth: Auth = Container.get(Auth);
-const authfbtoken: AuthFbToken = Container.get(AuthFbToken);
+
+$auth.registerUserRegistrationCallback((userAuthentication: IUserAuthentication) => {
+  console.log('USER REGISTERED', JSON.stringify(userAuthentication, null, 2));
+})
+
+const $authProviderPassword = Container.get(AuthProviderPassword);
+const $authProviderOAuth = Container.get(AuthProviderOAuth);
+const $authProviderEmail = Container.get(AuthProviderEmail);
+
+$authProviderEmail.registerSendMailCallback((mail: IProofMailPayload) => {
+  console.log('SEND PROOF MAIL', JSON.stringify(mail, null, 2));
+});
+
 const $email: NotificationsEmail = Container.get(NotificationsEmail);
 const $events: EventEmitter = Container.get(EventEmitter);
 const $dbAutoScaling: DbAutoScaling = Container.get(DbAutoScaling);
 
-$auth.setNotificationFunction(async (user, caller, meta) => {
+/* $auth.setNotificationFunction(async (user, caller, meta) => {
   console.log("> NOTIFY!", user.userId, caller, meta);
```

Comment:
This method does not exist anymore. So it should be removed instead of commented out. 

However, since this is just an example, I think it is not that important.